source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,94 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+``JsonObject`` is a more performant alternative to both ``Document``
```

Comment:
This claim is only true if you're only interested in using the JSON representation of the document, and not a Map. Let's avoid claims about performance, especially ones that depend on specific conditions, in the documentation if possible. Instead you can reword it to still mention it can have advantages, e.g.
"If you only want to work with JSON data, you can use JsonObject to avoid unnecessary data conversion to a Map"

Reply:
You can also mention that it's more efficient to avoid the data conversion, but performance and efficiency are actually different.

Reply:
You can also mention that it's more efficient to avoid the data conversion, but performance and efficiency are actually different. The former implies some controlled measurement and the latter is related to the number of and complexity of operations.

Reply:
Thanks for the feedback. I updated the section to use the new language here: https://github.com/mongodb/docs-java/pull/57/commits/4c9a8eba6f681feb0f9a2c6510b812f85914e565

Takeaways:

Be careful of using the word 'Performant' as it has a precise meaning requiring detailed measurement. Efficiency is generally a better word for our purposes as it has to do with number of code operations.